app/src/test/java/org/mozilla/fenix/library/bookmarks/BookmarkStoreTest.kt

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.library.bookmarks
+
+import kotlinx.coroutines.runBlocking
+import mozilla.components.concept.storage.BookmarkNode
+import mozilla.components.concept.storage.BookmarkNodeType
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotSame
+import org.junit.Test
+
+class BookmarkStoreTest {
+
+    @Test
+    fun `change the tree of bookmarks starting from an empty tree`() = runBlocking {
```

Comment:
Missing tests that cover `Change(newTree)` action:
- when `existingTree != null`
- when `existingTree==newTree`.

Reply:
Done.